src/test/web-platform-tests/converted/resources/cross-origin-helper-frame.js

```diff
@@ -1,6 +1,6 @@
 import "../../wpt-env.js";
 
-let cursor,db,store,value;
+let cursor,db,result,store,value;
```

Comment:
The newly-passing test has a strict mode bug where it sets a global `result` variable.